thirdeye/thirdeye-frontend/app/helpers/utils.js

```diff
@@ -51,6 +63,15 @@ export function checkStatus(response, mode = 'get', recoverBlank = false) {
   }
 }
 
+/**
+ * Converts the color string into its rgba cunterpart
```

Comment:
nice typo. lets delete the method if its not used - we can keep the mapping const

Reply:
Oops haha